### ISSUE TITLE
Connectkit SetOpen modal api not working

### ISSUE THREAD

**Issue by 0xTranqui** [mutualsupply/blog#13]
tldr we are using the connectkit api for accessing the modal functionality directly in the `handleMintInteraction` function used on the mint pages (aka `[listing].ts`) to trigger a connect wallet pop up if no one is logged in when a user hits the mint button. this was working great until a week ago, when it suddenly stopped working. accessing the connectkit stuff directly through the `connect` button in the bottom right corner works great.

@salieflewis could you try messing around with and see if your seeing the same thing?

<img width="273" alt="Screenshot 2023-05-28 at 10 45 58 PM" src="https://github.com/mutualsupply/blog/assets/93691906/2273f8ed-5a28-434e-9217-4b98f09468ac">
<img width="470" alt="Screenshot 2023-05-28 at 10 46 14 PM" src="https://github.com/mutualsupply/blog/assets/93691906/5c935efb-1ea8-4c0a-a8e9-b8fc4c665abe">

**Comment by 0xTranqui** [comment]
cc @gainor cuz why not

**Comment by salieflewis** [comment]
Pretty sure [setOpen](https://docs.family.co/connectkit/api-reference#useModal-setOpen) just accepts a boolean argument, and that is why it is not working.

`setOpen(true)`

**Comment by 0xTranqui** [comment]
you were 100% correct thank u sm! just fixed here https://github.com/mutualsupply/blog/pull/16.
